src/particle/particle/particle.py

```diff
@@ -1269,7 +1255,7 @@ def _from_group_dict_list(cls: Type[Self], mat: Dict[str, Any]) -> List[Self]:
         if mat["state"] is not None:
             kw["J"] = float(mat["state"])
 
-        maxname = name + "({mat[mass]})".format(mat=mat) if mat["mass"] else name
+        maxname = name + f"({mat['mass']})" if mat["mass"] else name
```

Comment:
```suggestion
        maxname = name + f"({mat["mass"]})" if mat["mass"] else name
```

Reply:
Content-free suggestion. Purely for consistency.

Reply:
The one directly above uses single on the outside and double in the middle.

Reply:
Oh boy, I was far too quick. Of course. Think so too for outer string single quoted.